cfme/intelligence/reports/reports.py

```diff
@@ -363,15 +323,43 @@ def data(self):
             return SavedReportData(headers, body)
 
     def download(self, extension):
-        self.navigate()
+        navigate_to(self, "Details")
         extensions_mapping = {'txt': 'Text', 'csv': 'CSV', 'pdf': 'PDF'}
         try:
             download_btn("Download as {}".format(extensions_mapping[extension]))
         except:
             raise ValueError("Unknown extention. check the extentions_mapping")
 
 
-class CannedSavedReport(CustomSavedReport):
+@navigator.register(CustomSavedReport, 'All')
+class SavedAll(CFMENavigateStep):
+    prerequisite = NavigateToAttribute('appliance', 'LoggedIn')
+
+    def step(self):
+        from cfme.web_ui.menu import nav
+        nav._nav_to_fn('Cloud Intel', 'Reports')(None)
+        accordion.tree("Reports", "All Reports")
+
+
+@navigator.register(CustomSavedReport, 'Details')
+class SavedDetails(CFMENavigateStep):
+    prerequisite = NavigateToSibling('All')
```

Comment:
Just looking at this, the prerequisite here could be to call the one from above like this

```python
def prerequisite(self):
    self.navigate_obj.navigate_to(CustomReport, 'All')
```

This then means you can drop the 'SavedAll' step above completely